Fix documentation for Command line (#27938)
home-assistant/home-assistant.github.io: source/_integrations/command_line.markdown

          required: false
          type: integer
          default: 60
        device_class:
          description: Sets the class of the device, changing the device state and icon that is displayed on the UI (see below). It does not set the `unit_of_measurement`.
          required: false
          type: device_class
        state_class:

>           description: "The [state_class](https://developers.home-assistant.io/docs/core/entity/sensor#available-state-classes) of the sensor. This will also display the value based on the user profile Number Format setting and influence the graphical presentation in the history visualization as a continuous value."

Reviewer: ```suggestion
          description: "The [state_class](https://developers.home-assistant.io/docs/core/entity/sensor#available-state-classes) of the sensor. This will display the value based on the **Number Format** defined in the user profile.  It will also influence the graphical presentation in the history visualization as a continuous value."
```
Not sure I understand the graphical presentation section. Does it mean "It changes the graphical presentation of the history visualization to show a continuous value"? A continuous value instead of what?

Author: Good question. I just did a copy/paste from Scrape without considering what is said really.
I'm fine if we should remove the last part.
What you think?

Reviewer 2: I would say remove it :)
